buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -22,12 +22,12 @@ object Dependencies {
     const val hyperionBuildConfig = "com.willowtreeapps.hyperion:hyperion-build-config:0.9.34"
     const val hyperionAppInfo = "com.star-zero:hyperion-appinfo:2.0.0"
     const val chucker = "com.github.ChuckerTeam.Chucker:library:3.5.2"
```

Comment:
zależało mi na podbiciu wersji Chucekra bo ma stare wersje libek od lifecycle ale tam jeszcze wisi na review update wersji wiec tylko to wrzucam